### ISSUE TITLE
Implement `flake8-raise`

### ISSUE THREAD

**Issue by ngnpope** [charliermarsh/ruff#2347]
[GitHub](https://github.com/jdufresne/flake8-raise), [PyPI](https://pypi.org/project/flake8-raise/).

- [x] [`R100`](https://github.com/jdufresne/flake8-raise#r100-raise-in-except-handler-without-from): raise in except handler without from
  - Already implemented via `B904` from [`flake8-bugbear`](https://github.com/charliermarsh/ruff#flake8-bugbear-b)
  - Already implemented via `TRY200` from [`tryceratops`](https://github.com/charliermarsh/ruff#tryceratops-try)
    - _This is buggy and only triggers with `raise Exception()` and not `raise Exception` (no parentheses)_
  - _Maybe implementation for one of these should be scrapped with the error code aliased to the other?_ :thinking:
- [x] [`R101`](https://github.com/jdufresne/flake8-raise#r101-use-bare-raise-in-except-handler): use bare raise in except handler
  - Already implemented via `TRY201` from [`tryceratops`](https://github.com/charliermarsh/ruff#tryceratops-try)
- [ ] [`R102`](https://github.com/jdufresne/flake8-raise#r102-unnecessary-parentheses-on-raised-exception): unnecessary parentheses on raised exception

So this plugin is mostly implemented. `R102` looks like it should be a relatively easy rule to define.

**Comment by charliermarsh** [comment]
Just confirming that in addition to adding R102, we should:

1. Remove `B904`, and mark it as an alias of `TRY200`.
2. Fix `TRY200` to flag `raise Exception` (without parentheses).

**Comment by ngnpope** [comment]
Yeah, sounds good. I guess it makes sense to keep `TRY200` over `B904` as `tryceratops` is exception-focused and `flake8-bugbear` is a random collection of stuff.
